content/release-2024-0/streaming-analytics-2024-0-bundle/2024_0.md

```diff
@@ -4,6 +4,10 @@ title: Release 2024.0
 layout: redirect
 ---
 
+### Change in basic diagnostics information
+
+The EPL memory profiler snapshots, which were previously included in the enhanced diagnostics information, have been moved to the basic diagnostics information. This is helpful in case a high memory usage alarm is raised when the Apama-ctrl microservice consumes 90% of the maximum memory permitted for the microservice container. See also [Downloading diagnostics and logs](https://cumulocity.com/guides/10.19.0/streaming-analytics/troubleshooting/#diagnostics-download) in the *Streaming Analytics* guide.
```

Comment:
I am thinking this still gives an impression that they are no longer available in enhanced diagnostics.

Maybe we can say : 
The EPL memory profiler snapshots, which were previously included only in the enhanced diagnostics information, are now also included in the basic diagnostics information. This is helpful in case a high memory usage alarm is raised when the Apama-ctrl microservice consumes 90% of the maximum memory permitted for the microservice container and we only have basic diagnostics information available.

Reply:
Oh, you're so right. Thanks :-) I've corrected this. But I used "you" instead of "we" in the additional text - ok?

Reply:
perfect, looks good now